mlflow/sagemaker/__init__.py

```diff
@@ -1692,11 +1696,14 @@ def _update_sagemaker_endpoint(
     # Create the new endpoint configuration and update the endpoint
     # to adopt the new configuration
     new_config_name = _get_sagemaker_config_name(endpoint_name)
-    # This is the hardcoded config for endpoint
+    config_tags = [{"Key": "app_name", "Value": endpoint_name}]
```

Comment:
This is duplicated code from above used in a similar context, should we break this out into it's own private function? Maybe `_get_sagemaker_config_tags`?